frontend/src/lib/components/ViewRecordingButton/ViewRecordingTrigger.tsx

```diff
@@ -64,8 +66,35 @@ export default function ViewRecordingTrigger({
             )
         }
     }
-    const disabledReason = sessionId ? undefined : 'No session ID provided'
+    const disabledReason = recordingDisabledReason(sessionId, recordingStatus)
     const link = inModal ? undefined : urls.replaySingle(sessionId ?? '')
 
     return children(onClick, link, disabledReason, maybeUnwatchedIndicator)
 }
+
+const recordingDisabledReason = (
+    sessionId: string | undefined,
+    recordingStatus: string | undefined
+): JSX.Element | string | null => {
+    if (!sessionId) {
+        return (
+            <>
+                No session ID associated with this event.{' '}
+                <Link to="https://posthog.com/docs/data/sessions#automatically-sending-session-ids">Learn how</Link> to
+                set it on all events.
+            </>
+        )
+    } else if (recordingStatus && !['active', 'sampled', 'buffering'].includes(recordingStatus)) {
+        return (
+            <>
+                Replay was not active when capturing this event.{' '}
+                <Link to="https://posthog.com/docs/session-replay/troubleshooting#recordings-are-not-being-captured">
+                    Learn why
+                </Link>{' '}
+                not all recordings are captured.
```

Comment:
@PostHog/team-replay heads up that I'm linking to this source when a recording isn't captured (most likely because the status is "disabled"). Let me know if there's somewhere better to send customers to get an explanation

Reply:
We can add a special page if it comes to that...